OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs

```diff
@@ -112,6 +115,10 @@ public Actor[] SendParatroopers(Actor self, WPos target, bool randomize = true,
 					});
 				}
 
+				if (!aircraftInRange.Any(kv => kv.Value))
```

Comment:
Play the notifications when the aircraft isn't in range of the lz?

Reply:
This is false when the first aircraft enters the area - it gets set to true just below.